package.json

```diff
@@ -40,16 +38,22 @@
     "url": "git://github.com/mozilla/shield-studies-addon-template.git"
   },
   "scripts": {
-    "eslint": "grunt eslint",
+    "eslint": "eslint --quiet **/*.js",
     "lint": "npm-run-all lint:*",
-    "lint:addons-linter": "# `addons-linter` will be caught during `test` # grunt shell:addonLintTest",
+    "lint:addons-linter": "jpm xpi && addons-linter shield-studies-addon-template.xpi",
     "lint:depcheck": "depcheck-ci  # use coverage to catch missing",
     "lint:eslint": "eslint .",
     "lint:fixpack": "fixpack",
     "lint:nsp": "nsp check",
     "prepublish": "npm shrinkwrap",
     "pretest": "npm-run-all lint:*",
-    "test": "grunt test && istanbul check-coverage --statements 100 --functions 100 --branches 100 --lines 100 coverage/reports/coverage.json"
+    "test": "npm run test:clean-coverage && npm run test:instrument && npm run test:jpm-test && npm run test:coverage-report",
+    "test:clean-coverage": "rm -rf coverage",
+    "test:coverage-report": "echo \"TODO: transpose grunt coverageReport here\"",
+    "test:ensure-files-are-covered": "./scripts/ensure-files-are-covered",
+    "test:instrument": "./scripts/instrument",
+    "test:jpm-test": "npm run test:ensure-files-are-covered && npm run test:make-test-env && cd testing-env && node_modules/.bin/jpm test --tbpl -b Aurora",
```

Comment:
Since the `jpm` module is installed locally (see L20 above), you can possibly remove the _./node_modules/.bin/_ prefix here since npm should automatically add that path when you're running via `npm run <script>`.